spec/_attachments/ic.did

```diff
@@ -241,5 +258,8 @@ service ic : {
     sender_canister_version : opt nat64;
   }) -> (record {canister_id : canister_id});
   provisional_top_up_canister :
-    (record { canister_id: canister_id; amount: nat }) -> ();
+    (record { canister_id : canister_id; amount: nat }) -> ();
+
+  // canister logging
+  fetch_logs : (record { canister_id : canister_id }) -> (fetch_logs_response) query;
```

Comment:
Just to double check: are we fine with the fact that the query results are not certified?

Reply:
Yes, I think we're fine with that.